### DISCUSSION TITLE
is a school setting up a github tutorial using bark?

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#116] [Q&A] [answered]
we've been getting a few forks with a couple or no changes.

**Comment by raynepaws** [comment, marked as answer]
i doubt it, bark is very small right now _(and i hate to be a debbie downer, but it might not go anywhere)_ so i don't think any state-commissioned schools/districts have even **heard** of bark from high above.

**Comment by mariocraft987** [comment]
Plus Bark isn't functional yet, they'd be better off using Scratch
